### ISSUE TITLE
esnext option doesn't appear to be working

### ISSUE THREAD

**Issue by kenwheeler** [jscs-dev/gulp-jscs#69]
Using the following syntax:
```
gulp.task('jscs', function(){
  return gulp.src('src/*.js')
      .pipe(jscs({
        esnext: true
      }))
});
```
I don't get any output. Running `jscs --esnext` on the CLI reports properly, and using the option with the `gulp-jscs-custom` module works though.

**Comment by LFDM** [comment]
Same here, unfortunately.

**Comment by dsebastien** [comment]
Same here

**Comment by geelen** [comment]
And here. Adding `esnext: true` changes the error from

```
    14 | fetch(window.location.origin + iceURL)
    15 |  .then(response => response.json())
--------------------------^
```

to

```
  line 15  col 24  'arrow function syntax (=>)' is only available in ES6 (use esnext option).
```

So clearly *something* is happening...

**Comment by geelen** [comment]
Whoops, it was a JSHint error that was running alongside! JSCS seems to be handling arrow functions fine.

**Comment by sindresorhus** [comment]
[Fixed in 3.0.0](https://github.com/jscs-dev/gulp-jscs/releases/tag/v3.0.0)

Specify it in your JSCS config file.
